x-pack/plugins/cross_cluster_replication/public/app/services/input_validation.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import Joi from 'joi';
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+
+export const i18nValidationErrorMessages = {
+  'any.empty': ({ label }) => (
+    i18n.translate('xpack.crossClusterReplication.formInputValidation.errorEmpty', {
+      defaultMessage: `{label} is required.`,
+      values: { label }
```

Comment:
I think we should check with the i18n team about this. We're essentially concatenating two individually localized strings, but I'm not sure if we can guarantee that will be semantically correct in all languages.

Reply:
it would be great to hear what they recommend for displaying a list of items/names within a sentence